science/gildas/Portfile

```diff
@@ -5,12 +5,12 @@ PortGroup           compilers 1.0
 PortGroup           active_variants 1.1
 
 name                gildas
-version             202409a
+version             202410a
 set my_version      [string tolower [clock format [clock scan 2000-[string range ${version} 4 5]-10] -format %b]][string range ${version} 2 3][string range ${version} 6 end]
 
 supported_archs     arm64 x86_64
 categories          science
-platforms           darwin
+# platforms           darwin
```

Comment:
please remove this line instead of commenting out

Reply:
Ok, done.